lemur/plugins/lemur_aws/tests/test_plugin.py

```diff
@@ -87,3 +87,79 @@ def test_upload_acme_token(app):
                                    options=additional_options,
                                    account_number=account)
     assert response
+
+
+@mock_sts()
+@mock_elb()
+@mock_ec2()
+@mock_elbv2()
+def test_get_all_elb_and_elbv2s(app, aws_credentials):
+    from copy import deepcopy
+    from lemur.plugins.lemur_aws.elb import get_load_balancer_arn_from_endpoint
+    from lemur.plugins.base import plugins
+    from lemur.plugins.utils import set_plugin_option
+
+    cert_arn_prefix = "arn:aws:iam::123456789012:server-certificate/"
+    cert_arn = cert_arn_prefix + "plugin-test.netflix.net-R3-20211023-20220121"
```

Comment:
nit; a more generic domain that is not company specific might better suited here. We have mainly used `example.com` in the past.

Reply:
Good point, done.